scipy/signal/tests/test_ltisys.py

```diff
@@ -233,7 +345,6 @@ def test_errors(self):
         assert_raises(ValueError, place_poles, A, B,
                       (-2+1j,-2-1j,-2+3j,-2-3j), method="KNV0")
 
-
```

Comment:
I don't know why/how I managed to remove this line... Thanks for pointing it out.